optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/META-INF/beans.xml

```diff
@@ -14,12 +14,11 @@
   ~ limitations under the License.
   -->
 
-<beans>
-  <scan>
-    <exclude name="**.client.**"/>
```

Comment:
Sure :) Some of the client classes were in jars that ended up in WEB-INF/lib (normally only server and shared jars should be in there) IIRC the errai jar had/has both server and client classes in the same jar (there was a good reason for it do so IIRC too).
Now when CDI scans the classpath (so the jars in WEB-INF/lib) it also scans those client classes: this is a performance loss on bootstrap, but also a theoretical risk to ambiguous injection errors, undesired alternative injections and whatnot - although those latter doesn't seem the case at all.